rikaicontent.js

```diff
@@ -321,19 +321,23 @@ var rcxContent = {
 	keysDown: [],
 	lastPos: { x: null, y: null},
 	lastTarget: null,
+	altGraphKey: false,
 
 	onKeyDown: function(ev) { rcxContent._onKeyDown(ev) },
 	_onKeyDown: function(ev) {
 		//console.log("key=" + ev.key + " keyCode=" + ev.keyCode + ' charCode=' + ev.charCode + ' detail=' + ev.detail);
  
+		if (ev.key === "AltGraph") {

```

Comment:
Should AltGr be treated as a separate key in the options or is it normal to treat it as identical to alt for these types of things?